catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -130,9 +133,14 @@
                             unbind-method="unbindPlugin" ref="resourceReaderSortedList"/>
 	</reference-list>
 
```

Comment:
@pklinef: Do you think we should use fix thread pools for both or just the `cacheThreadPool` like we have right now?